### PR TITLE
Support print log to file and console at the same time

### PR DESCRIPTION
**Pull request by wayblink**
related:#21936

Signed-off-by: wayblink <anyang.wang@zilliz.com>

### CODE REVIEW

**Review by LoveEachDay — APPROVED** [review]
/lgtm

File: internal/log/log.go [hunk 1]
```diff
@@ -68,23 +69,25 @@ func init() {
 
 // InitLogger initializes a zap logger.
 func InitLogger(cfg *Config, opts ...zap.Option) (*zap.Logger, *ZapProperties, error) {
-	var output zapcore.WriteSyncer
+	outputs := make([]zapcore.WriteSyncer, 0)
```
Comment (congqixia) [inline internal/log/log.go:72]: `var arr []T` is recommended comparing to `make ([]T, 0)`

File: internal/log/log.go [hunk 2]
```diff
@@ -129,7 +132,13 @@ func InitLoggerWithWriteSyncer(cfg *Config, output zapcore.WriteSyncer, opts ...
 
 // initFileLog initializes file based logging options.
 func initFileLog(cfg *FileLogConfig) (*lumberjack.Logger, error) {
-	if st, err := os.Stat(cfg.Filename); err == nil {
+	var logPath string
+	if len(cfg.RootPath) > 0 {
+		logPath = cfg.RootPath + string(filepath.Separator) + cfg.Filename
```
Comment (congqixia) [inline internal/log/log.go:137]: `path.Join()`?

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**